AdaptiveUICodelab/app/build.gradle

```diff
@@ -52,7 +52,7 @@ android {
         compose true
     }
     composeOptions {
-        kotlinCompilerExtensionVersion compose_version
+        kotlinCompilerExtensionVersion composeCompiler_version
```

Comment:
Nit: here its composeCompiler_version, the other one is compose_compiler_version